.luarc.json

```diff
@@ -5,5 +5,12 @@
     "param-type-mismatch",
     "missing-parameter"
   ],
-  "diagnostics.libraryFiles": "Disable"
+  "Lua.diagnostics.globals": [
```

Comment:
if you are having issues with these ( they get fixed by my PR)
before my PR:
<img width="771" alt="Screenshot 2022-10-14 at 3 23 57 PM" src="https://user-images.githubusercontent.com/10992695/195840780-312e9988-0fcb-4230-8a0a-36a4a81a3045.png">
after my PR:
<img width="834" alt="Screenshot 2022-10-14 at 3 23 19 PM" src="https://user-images.githubusercontent.com/10992695/195840815-54459422-2a3f-4b73-8692-4c5f1c669b3a.png">